docs/deploy_streamlit_app.md

```diff
@@ -12,19 +12,46 @@ Once you have your invite you're ready to deploy! It's really straightforward, j
 
 ## Put your Streamlit app on GitHub
 
-Make sure your app is in a public GitHub repo and that you have a [requirements.txt file](https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format)
+1. Add your Streamlit app to a public GitHub repo
+2. (recommended) Add a "requirements file" to manage any external dependencies
```

Comment:
Feels like we can remove the (recommended) note...it only helps beginners to feel like this is actually required